src/main/java/org/sagebionetworks/web/client/security/AuthenticationControllerImpl.java

```diff
@@ -215,8 +211,11 @@ public void onSuccess(String accessToken) {
           } else if (forceResetQueryClientCache) {
             resetQueryClientCache();
           }
-          userAccountService.getMyProfile(
-            new AsyncCallback<UserProfile>() {
```

Comment:
Tangentially related, I discovered this issue when I realized in my local stack sometimes this call would hang (in development only)-- replacing the RPC call to the servlet with a request sent directly via the JS client fixes that intermittent issue.